pkg/vdbgen/vdb.go

```diff
@@ -338,6 +371,23 @@ func (d *DBGenerator) setLocalPaths(ctx context.Context) error {
 	return nil
 }
 
+// setRequestSize will fetch the local data size and set it in v.vdb.
+func (d *DBGenerator) setRequestSize(ctx context.Context) error {
+	depotMaxSize, err := d.queryLocalDataSize(ctx, "DEPOT")
```

Comment:
I don't think we are handling the catalog in this function.  I wonder if we should pick the size of the disk where the depot, catalog, and data is hosted.  

You can do this via:
select max(disk_space_used_mb+disk_space_free_mb) from disk_storage where storage_usage = 'DEPOT';
select max(disk_space_used_mb+disk_space_free_mb) from disk_storage where storage_usage in ('CATALOG','DATA,TEMP');

Reply:
I was looking for a more accurate query. Thanks.